modules/configuration/README.md

```diff
@@ -109,6 +115,7 @@ public static class AbstractConfigurationSchema {
    a unique identifier among the inheritors of one polymorphic configuration, used to define the type (schema) of the polymorphic configuration we are dealing with now;
 * `@AbstractConfiguration` is similar to `@PolymorphicConfig` but its type cannot be changed and its inheritors must be annotated with
   either `@Config` or `@ConfigurationRoot`. Configuration schemas with this annotation cannot be used as a nested (sub)configuration;
+* `@ConfigurationExtension` allows to mix-in properties into `@Config` or `@ConfigurationRoot`.
```

Comment:
I don't quite understand what you wanted to say.
Probably it would be necessary to say that this is an extension and so on.